### DISCUSSION TITLE
国内公共模板无法更新的，请自定义加速源

### DISCUSSION THREAD

**Discussion by Kikyo18** [qd-today/qd#417] [General]
勾选默认仓库，然后点击退订。
再点击注册 仓库名随便写，仓库URL填写 `https://ghproxy.com/https://github.com/qd-today/templates` 分支 `master`  ， 加速选择关闭。
然后点重建缓存。
原仓库默认加速节点是jsdelivr，jsdelivr的国内节点基本上是gcore和cloudflare，联通用户基本连不上。最近电信用户走cloudflare的节点也经常性失联。

**Comment by a76yyyy** [comment]
现在支持在环境变量里配置加速源了, 具体参考 [文档](https://qd-today.github.io/qd/zh_CN/guide/deployment.html#%E9%85%8D%E7%BD%AE%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F)
